### PR TITLE
Create Api catch all with express

### PR DESCRIPTION
**Pull request by eladbash**

### CODE REVIEW

File: API/index.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import express = require('express');
+const app: express.Application = express();
+const port = 3000;
+
+app.get('*', (req: express.Request, res: express.Response) => {
+    res.send(`Called URL ${req.url}`);
+});
+
+app.listen(port, () => console.log(`Example app listening on port ${port}!`));
```
Comment (ChengaDev) [inline API/index.ts]: Change to WhiteTooth Server or something :)

File: API/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "start": "npm run build && node ./build/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc"
+  },
+  "author": "PizzaTip",
+  "license": "MIT",
+  "dependencies": {
+    "@types/express": "^4.17.2",
+    "express": "^4.17.1",
+    "typescript": "^3.7.5"
```
Comment (ChengaDev) [inline API/package.json]: for knowledge, TS is actually a package?

File: API/package.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "whitethoothapi",
+  "version": "1.0.0",
+  "description": "Mock API for teams with multi environment support",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run build && node ./build/index.js",
```
Comment (ChengaDev) [inline API/package.json]: maybe the user would be able to set the port through a parameter for to start command (with default)?

File: API/package.json (hunk 2 above)
Comment (eladbash) [reply API/package.json]: Yap

File: API/package.json (hunk 3 above)
Comment (eladbash) [reply API/package.json]: We will define dedicated task for all the startup of the full service

**Pull request merged**